validator/val_utils.c

```diff
@@ -560,7 +593,7 @@ val_verify_DNSKEY_with_DS(struct module_env* env, struct val_env* ve,
 	if(!has_useful_ds) {
 		verbose(VERB_ALGO, "No usable DS records were found -- "
 			"treating as insecure.");
-		return sec_status_insecure;
+			return sec_status_insecure;
```

Comment:
Accidentally added a tab?
```suggestion
		return sec_status_insecure;
```